### PR TITLE
[Cache] Rename nonce to version

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Branch? | 3.1 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | https://github.com/symfony/symfony-docs/pull/6515#discussion_r63296493 |
| License | MIT |
| Doc PR | - |

ping @javiereguiluz

### CODE REVIEW

File: src/Symfony/Component/Cache/Adapter/ApcuAdapter.php [hunk 1]
```diff
@@ -34,12 +34,12 @@ public function __construct($namespace = '', $defaultLifetime = 0, $nonce = null
...
-        if (null !== $nonce) {
-            CacheItem::validateKey($nonce);
+        if (null !== $version) {
+            CacheItem::validateKey($version);
 
-            if (!apcu_exists($nonce.':nonce'.$namespace)) {
+            if (!apcu_exists($version.':version'.$namespace)) {
                 $this->clear($namespace);
-                apcu_add($nonce.':nonce'.$namespace, null);
+                apcu_add($version.':version'.$namespace, null);
```
Comment (javiereguiluz) [inline src/Symfony/Component/Cache/Adapter/ApcuAdapter.php]: If `$namespace = 'my_namespace'` and `$version = 'v3'`, the we'll use `v3:versionmy_namespace`. Are we sure we don't want to do something like: `v3:my_namespace` or `v3:version:my_namespace` ?

Comment (nicolas-grekas) [inline src/Symfony/Component/Cache/Adapter/ApcuAdapter.php]: We just need a separator, the actual value is of no importance...

Comment (nicolas-grekas) [inline src/Symfony/Component/Cache/Adapter/ApcuAdapter.php]: `:` is now the separator

**Comment by javiereguiluz** [comment]
👍

**Pull request closed without merging**